starwars/src/components/StarWars.css

```diff
@@ -0,0 +1,15 @@
+.Styling {
+    background: yellow;
+    border: 3px solid blue;
+    font-weight: bold;
+    height: 100px;
+    width: 1500px;
```

Comment:
Don't forget your CSS responsive rules! Try to use percentages for widths, and a max-width property to prevent them from being too large on big screens.